source/common/phase_impl.h

```diff
@@ -0,0 +1,26 @@
+
+#pragma once
+
+#include "nighthawk/common/phase.h"
+
+#include "external/envoy/source/common/common/logger.h"
+
+namespace Nighthawk {
+
+class PhaseImpl : public Phase, public Envoy::Logger::Loggable<Envoy::Logger::Id::main> {
+public:
+  PhaseImpl(absl::string_view id, SequencerPtr&& sequencer, bool measure_latencies)
```

Comment:
The same comment about boolean flags applies here to `measure_latencies`. Would it be worth coming up with something more readable to replace it with?

Reply:
Done